release/release_tests.yaml

```diff
@@ -6017,48 +6017,65 @@
 
 - name: aws_cluster_launcher
   group: cluster-launcher-test
-  working_dir: ../python/ray/autoscaler/aws/
+  working_dir: ../python/ray/autoscaler/
 
   stable: true
 
   frequency: nightly
   team: core
   cluster:
-    cluster_env: tests/aws_config.yaml
-    cluster_compute: tests/aws_compute.yaml
+    cluster_env: aws/tests/aws_config.yaml
+    cluster_compute: aws/tests/aws_compute.yaml
 
   run:
     timeout: 1200
-    script: cd tests && python aws_launch_and_verify_cluster.py aws_cluster.yaml
+    script: python launch_and_verify_cluster.py aws/tests/aws_cluster.yaml
 
 - name: aws_cluster_launcher_minimal
   group: cluster-launcher-test
-  working_dir: ../python/ray/autoscaler/aws/
+  working_dir: ../python/ray/autoscaler/
 
   stable: true
 
   frequency: nightly
   team: core
   cluster:
-    cluster_env: tests/aws_config.yaml
-    cluster_compute: tests/aws_compute.yaml
+    cluster_env: aws/tests/aws_config.yaml
+    cluster_compute: aws/tests/aws_compute.yaml
 
   run:
     timeout: 1200
-    script: cd tests && python aws_launch_and_verify_cluster.py ../example-minimal.yaml
+    script: python launch_and_verify_cluster.py aws/example-minimal.yaml
 
 - name: aws_cluster_launcher_full
   group: cluster-launcher-test
-  working_dir: ../python/ray/autoscaler/aws/
+  working_dir: ../python/ray/autoscaler/
 
   stable: true
 
   frequency: nightly
   team: core
   cluster:
-    cluster_env: tests/aws_config.yaml
-    cluster_compute: tests/aws_compute.yaml
+    cluster_env: aws/tests/aws_config.yaml
+    cluster_compute: aws/tests/aws_compute.yaml
 
   run:
     timeout: 1200
-    script: cd tests && python aws_launch_and_verify_cluster.py ../example-full.yaml
+    script: python launch_and_verify_cluster.py aws/example-full.yaml
+
+- name: gcp_cluster_launcher_minimal
+  group: cluster-launcher-test
+  working_dir: ../python/ray/autoscaler/
+
+  stable: true
+
+  env: gce
+  frequency: nightly
+  team: core
+  cluster:
+    cluster_env: gcp/tests/gce_config.yaml
+    cluster_compute: gcp/tests/single_node_32_cpu_gce.yaml
+
+  run:
+    timeout: 1200
+    script: python launch_and_verify_cluster.py gcp/example-minimal.yaml
```

Comment:
I imagine there was some amount of manual tweaking of permissions in our release testing clouds to get this to work, or are there any assumptions that we should make sure are written down about how the cloud needed to be set up in order for this test to run?

Reply:
It ended up not needing any tweaking apart from removing a subnet that was added for someone's ad hoc load balancer testing in the past that it was somehow defaulting to.

We're using the same cloud that's used to run release tests on GCP (we grab the current GCP project from the release test machine and inject it into the example YAML as the project ID), and I guess the assumption is that it already has the appropriate permissions to create and delete nodes because it's the cloud used for release tests (so it has to start up and spin down clusters).